cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -44,6 +45,8 @@ const (
 	operationWaitTimeout    = 5 * time.Second
 	operationPollInterval   = 100 * time.Millisecond
 	maxRecordsReturnedByAPI = 100
+	maxRetryDeadline        = 5 * time.Minute
```

Comment:
As discussed in-person, let's please make it smaller, may be 1 minute ?

Reply:
Done. Refer here - https://github.com/gardener/autoscaler/pull/11/commits/01144f3ab8d273f0eb3ad652cd86e095ad0afee8#diff-50934888f33c34f8e25393b844c0c8c6R48-R49